src/app/container/state/refresh-wizard.service.ts

```diff
@@ -18,7 +17,6 @@ export class RefreshWizardService {
   constructor(
     private refreshWizardStore: RefreshWizardStore,
     private dockerRegistriesService: UsersService,
```

Comment:
It was like this, but I suggest naming the property `usersService`; I was initially confused reading the changes below.